py3status/constants.py

```diff
@@ -10,6 +10,11 @@
     "output_format": "i3bar",
 }
 
+DEFAULT_SEPARATORS = {
+    "i3bar": None,
```

Comment:
I think the code should be smart enough to not needing to have `i3bar` and others here.  i.e `get()`. Also, because we might have to consider more than just two. See below.

Reply:
agreed. I'll address this in the next commit.